packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,100 @@
+# Select
+
+## Component Description
+
+The browser-native select control component allows users to choose one option from a dropdown list. It is commonly used in forms for selecting values from a predefined set of options.
+
+## Design Spec
+
+[Link to Design Spec in Figma](https://www.figma.com/file/0vOH481cv1VQyfQIX9JALn/Select?node-id=1319-163&t=nn2xgT4ieHjbkqQW-11)
+
+## Engineering Spec
+
+### Inputs
+
+- @attr appearance: "outline" "underline" "filled-darker" "filled-lighter" | "outline"
+- @attr size: "small" "medium" "large" | "medium"
+- @attr disabled: boolean
+- @attr multiple: boolean
+- @attr required: boolean
+- @attr name: string
+- @attr autofocus: boolean
+- @attr autocomplete: "on" "off" | "off"
+- @attr aria-label: string
+- @attr aria-labelledby: string
+- @attr aria-describedby: string
+
+### Outputs
+
+None
+
+### Events
+
+- @input: Fires whenever the value of the `select` has been changed
+- @change: Fires whenever the user modifies the value of `select`
+
+### Slots
```

Comment:
I think we should consider a label slot until we get the cross root aria stuff figured out.

Reply:
If we have a label slot, does it make sense to use the wrap the slot in a native HTML `label` element, or should we use the `fluent-label` web component here?

Reply:
Great question - what are the implications with a slot vs. without one?

Reply:
> native HTML label element, or should we use the fluent-label web component

We should always weight what % of the custom component will be used in the particular scenario (vs what will be added to the bundle) and also how difficult would it be to recreate the component in place.
And imo the biggest factor is - when we will change `fluent-label`, would you expect the label in `fluent-select` to change as well?
In this case, we will perhaps use most of the `fluent-label` functionality, the select's label should be consistent with `fluent-label` so it is ok to use `fluent-label`.

> how do we feel about adding a label slot for this specific component when the other components don't? Is it inconsistent? Is that a bad thing?

Inconsistency is definitely a bad thing. But in this case, I think it is an temporary earned difference until we have cross root aria.